### PR TITLE
Solve: All

### PR DESCRIPTION
**Pull request by criticic**
Not able to understand that part, and where exactly do I have to enter its solution.

Kindly guide.

### CODE REVIEW

**Comment by criticic** [comment]
also there is lot of inconsistency in weather a newline should be at end of output or not

**Comment by ishashankmittal** [comment]
> also there is lot of inconsistency in weather a newline should be at end of output or not

Yes, I didn't mentioned it explicitly, will update now. I would have mentioned \n character in the desired output if wanted a new line.

**Comment by ishashankmittal** [comment]
> Not able to understand that part, and where exactly do I have to enter its solution.
> 
> Kindly guide.
The goroutine tasks needs to be done in main.go file, you can delete the existing function calls.

**Comment by ishashankmittal** [comment]
@criticic  Also share the screen shot of the terminal after running ```go test ./...``` so that i can see your score.

File: main.go
Comment (ishashankmittal) [inline main.go:1]: implement the goroutines task here.

**Comment by criticic** [comment]
image

for some reason I am not getting the score anymore, utilll it was like 36 I got the score, but as soon as I did the  `CheckIfSumIsLessThan10()` (last one) im not getting the score

**Comment by ishashankmittal** [comment]
> image
> for some reason I am not getting the score anymore, utilll it was like 36 I got the score, but as soon as I did the `CheckIfSumIsLessThan10()` (last one) im not getting the score

Since you have passed all the tests, the TestMain function returns ok, this is just how the tests were written.
Ok means you have passed all the tests.

**Comment by criticic** [comment]
it took me quite a while to realise that if the program exits the goroutines wont run at all😅, anyway @ishashankmittal kindly check now

File: main.go
Comment (criticic) [reply main.go:1]: Done

**Comment by ishashankmittal** [comment]
> it took me quite a while to realise that if the program exits the goroutines wont run at all😅, anyway @ishashankmittal kindly check now

Reviewed your Goroutines task. Nice to see you figured out to use time.Sleep in the main(). This is where sync and wait groups come into play.

**Comment by ishashankmittal** [comment]
Good Work @criticic , GG
Closing the PR for now, don't forget to ping me on whatsapp.

**Pull request closed without merging**